eosdis-nasa/earthdata-pub-api
# EDPUB1181: Update cloud metrics logs

# Description

Update log of `workflow_complete` to include `time_to_publish` seconds rounded (from milliseconds)

## Spec

See Ticket: https://bugs.earthdata.nasa.gov/browse/EDPUB-1181

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Push related branches to SIT
3. Run a request to completion
4. Confirm in the metrics lambda logs that you see a log of the event with event_type = `workflow_completed` and which has an attribute for `time_to_publish` and an integer value
